docs/mkdocs/docs/api/macros/nlohmann_json_serialize_enum _strict.md

```diff
@@ -0,0 +1,93 @@
+# NLOHMANN_JSON_SERIALIZE_ENUM_STRICT
+
+```cpp
+#define NLOHMANN_JSON_SERIALIZE_ENUM_STRICT(type, conversion...)
+```
+
+The `NLOHMANN_JSON_SERIALIZE_ENUM_STRICT` allows to define a user-defined serialization for every enumerator.
+
+This macro declares strict serialization and deserialization functions (`to_json` and `from_json`) for an enum type. 
+Unlike [`NLOHMANN_JSON_SERIALIZE_ENUM`](nlohmann_json_serialize_enum.md), this macro enforces strict validation and 
+throws errors for unmapped values instead of defaulting to the first enum value.
+
+## Parameters
+
+`type` (in)
+:   name of the enum to serialize/deserialize
+
+`conversion` (in)
+:   a pair of an enumerator and a JSON serialization; arbitrary pairs can be given as a comma-separated list
```

Comment:
I would say "pairs" rather than "a pair", and "a JSON serialization" is odd, maybe "the string to be used in the JSON serialization.  The "arbitrary pairs can be given as a comma-separated list" sounds weird to me.
Maybe
> A list of parameters alternating between an enumerator value and a string to use in the JSON serialization.